product_state/__manifest__.py

```diff
@@ -2,12 +2,12 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
     'name': "Product State",
-    'description': """
+    'summary': """
```

Comment:
@Nikul-Chaudhary Could you use README generator ?

Reply:
@rousseldenis @OCA-git-bot  is automatically generated.

Reply:
@Nikul-Chaudhary only if you use README fragments

Reply:
@rousseldenis Done